yb-voyager/src/srcdb/postgres.go

```diff
@@ -60,7 +60,8 @@ func GetPGLiveMigrationUnsupportedDatatypes() []string {
 }
 
 func GetPGLiveMigrationWithFFOrFBUnsupportedDatatypes() []string {
-	unsupportedDataTypesForDbzmYBOnly, _ := lo.Difference(YugabyteUnsupportedDataTypesForDbzm, PostgresUnsupportedDataTypes)
+	//TODO: add connector specific handling 
```

Comment:
I don't think we can at this point? 
If i understand correctly, `GetPGLiveMigrationWithFFOrFBUnsupportedDatatypes` is called by assess-migration. At this point, we don't know which connector they are going to use, so we can't really make that call. Correct?

Reply:
yeah